xmppserver/src/main/java/org/jivesoftware/openfire/muc/spi/MultiUserChatServiceImpl.java

```diff
@@ -1281,49 +1306,75 @@ public void initializeSettings() {
         }
     }
 
-    public void setLogMaxConversationBatchSize( int size )
+    @Override
+    public void setLogMaxBatchSize( int size )
```

Comment:
Here, the public API was renamed (from `setLogMaxConversationBatchSize` to `setLogMaxBatchSize`). It's _probably_ fine, but there's a risk that we're breaking something (eg: third party plugin) with this change. Would preventing the rename introduce an issue elsewhere?

Reply:
I'll revert this. The naming is slightly inconsistent but that's not worth upsetting third parties for.